internal/core/tests.rs

```diff
@@ -61,32 +61,45 @@ pub extern "C" fn slint_send_mouse_click(
 #[no_mangle]
 pub extern "C" fn send_keyboard_string_sequence(
     sequence: &crate::SharedString,
-    modifiers: KeyboardModifiers,
     window_adapter: &crate::window::WindowAdapterRc,
 ) {
     for ch in sequence.chars() {
-        let mut modifiers = modifiers;
         if ch.is_ascii_uppercase() {
-            modifiers.shift = true;
+            WindowInner::from_pub(window_adapter.window()).process_key_input(KeyInputEvent {
+                event_type: KeyEventType::KeyPressed,
```

Comment:
and it is never release?

I wonder if we cannot just remove this hack anyway.  if tests passes without it it can be removed.

Reply:
the whole test stuff is wip.